src/rgw/radosgw-admin/radosgw-admin.cc

```diff
@@ -146,6 +145,12 @@ void usage()
   cout << "  user policy list attached        list attached managed policies\n";
   cout << "  caps add                         add user capabilities\n";
   cout << "  caps rm                          remove user capabilities\n";
+  cout << "  dedup stats                      Collcet & display dedup statistics\n";
+  cout << "  dedup abort                      Abort dedup\n";
+  cout << "  dedup restart dry                Restart dedup (dry run)\n";
```

Comment:
not critical, but would be nice if `--dry` was a parameter to the `restart` command, and not a sub command.
as an example of how to use a no-value (=binary) flag, you can use `--commit` as an example:
```
 } else if (ceph_argparse_binary_flag(args, i, &commit, NULL, "--commit", (char*)NULL)) {
      // do nothing
 }
```
note that this should also be added to the "usage" print, for example, see line 525 with command specific options.
and, this would also require a change to `help.t`...